enterprise/cli/provisionerdaemonstart.go

```diff
@@ -136,12 +151,17 @@ func (r *RootCmd) provisionerDaemonStart() *serpent.Command {
 				logger.Info(ctx, "note: untagged provisioners can only pick up jobs from untagged templates")
 			}
 
-			// When authorizing with a PSK, we automatically scope the provisionerd
-			// to organization. Scoping to user with PSK auth is not a valid configuration.
+			// When authorizing with a PSK / provisioner key, we automatically scope the provisionerd
+			// to organization. Scoping to user with PSK / provisioner key auth is not a valid configuration.
 			if preSharedKey != "" {
-				logger.Info(ctx, "psk auth automatically sets tag "+provisionersdk.TagScope+"="+provisionersdk.ScopeOrganization)
+				logger.Info(ctx, "psk automatically sets tag "+provisionersdk.TagScope+"="+provisionersdk.ScopeOrganization)
 				tags[provisionersdk.TagScope] = provisionersdk.ScopeOrganization
 			}
+			if provisionerKey != "" {
+				logger.Info(ctx, "provisioner key auth automatically sets tag "+provisionersdk.TagScope+" empty")
+				// no scope tag will default to org scope
+				delete(tags, provisionersdk.TagScope)
+			}
```

Comment:
Is this because it's all done on coderd? And we should be passing no tags into the provisioner daemon?

Reply:
Correct, the API will actually fail if you pass any tags + a provisioner key. I felt this was a good opportunity to break that behavior altogether.